jest.config.js

```diff
@@ -0,0 +1,18 @@
+const packageJSON = require('./package.json');
+
+process.env.TZ = 'UTC';
+
+module.exports = {
+  verbose: true,
+  name: packageJSON.name,
+  displayName: packageJSON.name,
+  transform: {
+    '\\.[jt]sx?$': 'babel-jest',
+  },
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
+  testURL: 'http://localhost',
+  transformIgnorePatterns: ['[/\\\\]node_modules[/\\\\].+\\.(js|jsx)$'],
+  testMatch: ['**/*.(spec|test).js'],
```

Comment:
I've used spec as a postfix on the files because IMO is easy to filter tests from other files without looking at the path.